### PR TITLE
Adds option to silence request params logging

### PR DESCRIPTION
**Pull request by darbyfrey**
@bellycard/platform @bellycard/data 

Since this is an ENV that would make it easy to on/off as needed

### CODE REVIEW

File: lib/napa/middleware/logger.rb [hunk 1]
```diff
@@ -38,16 +38,17 @@ def format_request(env)
...
-          remote_ip:        request.ip
-        }
+        request_data = {}.tap do |h|
+          h[:method]      = request.request_method
+          h[:path]        = request.path_info
+          h[:query]       = filtered_query_string(request.query_string)
+          h[:host]        = Napa::Identity.hostname
+          h[:pid]         = Napa::Identity.pid
+          h[:revision]    = Napa::Identity.revision
+          h[:params]      = params unless ENV['SILENCE_REQUEST_PARAMS_LOG'] == 'true'
```
Comment (ashtonthomas) [inline lib/napa/middleware/logger.rb]: Do you want to bypass the `filtered_parameters` here to show everything or did you have trouble with failing specs? For the latter see https://github.com/bellycard/napa/pull/258

**Comment by ashtonthomas** [comment]
:+1: to removal of gem_dependency
:+1: to `SILENCE_REQUEST_PARAMS_LOG `

one comment

File: lib/napa/middleware/logger.rb (hunk 1 above)
Comment (darbyfrey) [inline lib/napa/middleware/logger.rb]: Good catch! Yea, that was a mistake, I've updated now.

**Comment by ashtonthomas** [comment]
+ 1

**Comment by ashtonthomas** [comment]
+1

**Pull request merged**